Src/bootloader.c

```diff
@@ -135,7 +135,7 @@ void BL_ReadMemory_Command(uint32_t address, uint8_t nob, uint8_t *pData)
   uint8_t addr_frame[5];
   uint8_t nob_frame[2];
   uint8_t dummy = 0x00U;
-
+  uint16_t nobrx = nob+1;

```

Comment:
I think it would be easier to understand from an API point of view if the nob parameter was the number of Bytes to be read and not nob-1. What about changing the nob parameter to a `uint16_t` instead?